aeron/ringbuffer/manytoone.go

```diff
@@ -93,7 +95,7 @@ func (buf *ManyToOne) consumerPosition() int64 {
 	return buf.buffer.GetInt64Volatile(buf.headPositionIndex)
 }
 
-func (buf *ManyToOne) claimCapacity(requiredCapacity int32) int32 {
+func (buf *ManyToOne) claimCapacity(requiredCapacity int32) (int32, error) {
```

Comment:
I think we should revert this. insufficient capacity isn't an error. the -2 matches the C++/Java https://github.com/real-logic/aeron/blob/master/aeron-client/src/main/cpp/concurrent/ringbuffer/ManyToOneRingBuffer.h#L243

Reply:
Done, here and with NullCounterId elsewhere (in keeping with the same theme)